src/controllers/MLFacetQueryController.ts

```diff
@@ -19,6 +20,10 @@ export class MLFacetQueryController {
     this.numberOfValuesToRequest = this.facet.options.numberOfValues;
   }
 
+  public setFreezeCurrentValuesFlag(freezeCurrentValues: boolean) {
```

Comment:
That's kind of a useless method.
Why not make the property public ?

There's no validation or "additional" work that needs to be performed when the flag is set.